docs/user/commands/kibana-verification-code.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[kibana-verification-code]]
+=== kibana-verification-code
+
+The `kibana-verification-code` tool generates a verification code for enrolling
```

Comment:
Technically it's not generating a verification code but retrieving it. Every time you run it you will receive the same code until you restart Kibana which would then generate a new one.